dev/NavigationView/NavigationView.cpp

```diff
@@ -365,14 +365,23 @@ void NavigationView::OnApplyTemplate()
     if (SharedHelpers::IsThemeShadowAvailable())
     {
 #ifdef USE_INSIDER_SDK
-        if (auto splitView = m_rootSplitView.get())
+        // Shadow will get clipped if casting on the splitView.Content directly
+        // Creating a canvas with negative margins as receiver to allow shadow to be drawn outside the content grid 
+        winrt::Canvas shadowReceiver;
+        winrt::Thickness shadowReceiverMargin = { -OpenPaneLength(), -c_paneElevationTranslationZ, -c_paneElevationTranslationZ, -c_paneElevationTranslationZ };
+        shadowReceiver.Margin(shadowReceiverMargin);
```

Comment:
What is the reason for this specific value for margin? Why c_paneElevationTranslationZ?

Reply:
It's kind of "arbitrary", we need enough space to draw the shadow without any clip, I tried manually the number is around 30. Also, when elevation gets higher, we'll need more space since shadow will get longer, that variable seems to be a good fit...

I'll follow up with Dmitriy to see if there's a way to calculate exactly how much space we need.

Reply:
@DmitriyKomin

Reply:
Asked Dmitriy and seems like it's not straight forward to get the actual values...

Reply:
Ok, leave it hard coded if there is no alternative.